src/components/DataArchives/DataArchives.js

```diff
@@ -89,14 +89,15 @@ const styles = theme => ({
     marginTop: '2rem'
   }
 });
-const sensors = 'http://api.airquality.codeforafrica.org/v1/sensor/{sensorID}/';
-const query = 'http://api.airquality.codeforafrica.org/v1/filter/{query}';
-const data = 'http://api.airquality.codeforafrica.org/v1/data/';
-const now = 'http://api.airquality.codeforafrica.org/v1/now/';
+// const sensors = 'https://api.sensors.africa/v1/sensor/{sensorID}/';
```

Comment:
set this as `https://api.sensors.africa/v1/sensors/{sensor id}/`

test -> https://api.sensors.africa/v1/sensors/140/